### PR TITLE
New: Getting Started Page and Button (fixes #101)

### LINKED ISSUE #101: No clear install instructions
I was told by a coworker that we were moving to eslint, so I came to the website and scoured it looking for the big "install now!" button or some call-to-action... looked through the "configuring eslint" page, then tried the "about" page, then clicked the "latest news link" but the install instructions looked strange as it said it was a "pre-release" version... because I'd clicked on everything else I thought I'd give "Command Line Interface" a go, and for some reason that's where the install instructions were. 

Not a big deal, but I just thought I'd let you know my experience.

### PR DESCRIPTION
**Pull request by IanVS**
Mostly copied from the GitHub README, with a few tweaks and additions.

### CODE REVIEW

File: docs/getting-started/index.md [hunk 1]
```diff
@@ -0,0 +1,66 @@
...
+
+* ESLint uses [Espree](https://github.com/eslint/espree) for JavaScript parsing.
+* ESLint uses an AST to evaluate patterns in code.
+* ESLint is completely pluggable, every single rule is a plugin and you can add more at runtime.
+
+## Installation
+
+You can install ESLint using npm:
+
+    npm install -g eslint
```
Comment (ilyavolodin) [inline docs/getting-started/index.md]: I think this should be expended. Explaining local vs. global install, having ESLint as a dependency of your project, etc.

File: _includes/jumbotron.html [hunk 2]
```diff
@@ -7,5 +7,10 @@ <h2 itemprop="description">The pluggable linting <span itemprop="applicationCate
           </div>
           <div class="col-md-4 col-sm-4"></div>
         </div>
+        <div class="row">
```
Comment (ilyavolodin) [inline _includes/jumbotron.html]: I can't say I love the placement of that button, but I can't really come up with the alternative...

**Comment by nzakas** [comment]
@IanVS @ilyavolodin what's the status of this?

**Comment by IanVS** [comment]
I need to expand the section about installing globally vs. locally.  Other than that, I think it's ready, unless someone has better ideas.

**Comment by nzakas** [comment]
It doesn't have to be perfect in the first go-around. If it's good enough now, let's make it live and you can continue working on it. The number of issues related to this is growing, so I don't think we need to hold off for perfection.

File: docs/getting-started/index.md [hunk 3]
```diff
@@ -0,0 +1,66 @@
...
+
+## Usage
+
+If it's your first time using ESLint, you should set up a config file using `--init`:
+
+    eslint --init
```
Comment (Sequoia) [inline docs/getting-started/index.md]: :+1:

**Comment by IanVS** [comment]
OK, I'm fine with that.  Continuous deployment FTW!  :smiley_cat:

**Comment by nzakas** [comment]
Ok, let's do it

**Comment by Sequoia** [comment]
:clap: :clap: :+1:

**Pull request merged**